File: minseon/22_11_17_02.py
#삼각형의 완성조건 (1)

def solution(sides):
    answer = 0
    max_1 = int(max(sides))
    a,b,c = map(int,sides)
    if a == max_1:
        if b+c > a:
            answer = 1
        else:
            answer = 2
    elif b == max_1:
        if a+c > b:
            answer = 1
        else:
            answer = 2
    else:
        if a+b > c:
            answer = 1
        else:
            answer = 2
    return answer

File: seungbeom/22_11_17_02.java
// https://school.programmers.co.kr/learn/courses/30/lessons/120889

class Solution {
    public int solution(int[] sides) {
        int a = 0;
        int b = 0;
        int h = sides[0];
        if ( h < sides[1] ) {
            a = h;
            h = sides[1];
        } else {
            a = sides[1];
        }

        if ( h < sides[2] ) {
            b = h;
            h = sides[2];
        } else {
            b = sides[2];
        }

        return ( h < a + b ) ? 1 : 2;
    }
}